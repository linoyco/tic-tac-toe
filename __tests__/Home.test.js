import "react-native";
import React from "react";
import Home from "../src/Containers/HomePage";
import renderer from "react-test-renderer";

test("Home snapShot", () => {
    const snap = renderer.create(<Home />).toJSON();
    expect(snap).toMatchSnapshot();
});

