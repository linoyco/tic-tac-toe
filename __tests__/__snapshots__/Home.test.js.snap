// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapShot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "height": "80%",
      "justifyContent": "center",
      "margin": "10%",
      "padding": 15,
      "width": "80%",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 30,
        "fontWeight": "bold",
      }
    }
  >
    Welcome to
  </Text>
  <Text
    style={
      Array [
        Object {
          "fontSize": 30,
          "fontWeight": "bold",
        },
        Object {
          "marginBottom": "50%",
        },
      ]
    }
  >
    Tic Tac Toe!
  </Text>
  <View
    style={
      Object {
        "borderColor": "black",
        "borderRadius": 5,
        "borderWidth": 1,
        "padding": 0,
      }
    }
  >
    <View
      style={
        Object {
          "borderColor": "yellow",
          "borderRadius": 5,
          "borderWidth": 2,
          "padding": 0,
        }
      }
    >
      <View
        style={
          Object {
            "alignContent": "center",
            "alignItems": "center",
            "borderColor": "black",
            "borderRadius": 5,
            "borderWidth": 1,
            "flexDirection": "row",
            "justifyContent": "center",
            "padding": 10,
          }
        }
      >
        <Text />
        <Text
          style={
            Object {
              "fontSize": 30,
              "fontWeight": "bold",
              "marginLeft": "10%",
              "marginRight": "10%",
            }
          }
        >
          VS
        </Text>
        <Text />
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "marginTop": "auto",
        "width": "100%",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "width": "100%",
        }
      }
    >
      <View
        style={
          Object {
            "backgroundColor": "#6200ee",
            "borderColor": "transparent",
            "borderRadius": 4,
            "borderStyle": "solid",
            "borderWidth": 0,
            "elevation": 2,
            "marginBottom": "2%",
            "marginTop": "2%",
            "minWidth": 64,
            "shadowColor": "#000000",
            "shadowOffset": Object {
              "height": 0.75,
              "width": 0,
            },
            "shadowOpacity": 0.24,
            "shadowRadius": 1.5,
            "width": "100%",
          }
        }
      >
        <View
          accessibilityRole="button"
          accessibilityStates={Array []}
          accessible={true}
          focusable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              Object {
                "overflow": "hidden",
              },
              Object {
                "borderRadius": 4,
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "flexDirection": "row",
                  "justifyContent": "center",
                },
                undefined,
              ]
            }
          >
            <Text
              numberOfLines={1}
              style={
                Array [
                  Object {
                    "color": "#000000",
                    "fontFamily": "System",
                    "fontWeight": "400",
                    "textAlign": "left",
                  },
                  Array [
                    Object {
                      "letterSpacing": 1,
                      "marginHorizontal": 16,
                      "marginVertical": 9,
                      "textAlign": "center",
                    },
                    undefined,
                    Object {
                      "textTransform": "uppercase",
                    },
                    Object {
                      "color": "#ffffff",
                      "fontFamily": "System",
                      "fontWeight": "500",
                    },
                    Object {
                      "fontFamily": "System",
                      "fontWeight": "500",
                    },
                    undefined,
                  ],
                ]
              }
            >
              Please login
            </Text>
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;
